src/components/gifList/GifList.style.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const StyledGifList = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  .gif {
```

Comment:
create another component

Reply:
Hey Tim, thank you for your feedback, I updated as suggested.